### PR TITLE
BaSP-TG-40: List tasks

### PR DESCRIPTION
**Pull request by Gearyandres94**

### CODE REVIEW

File: src/Components/Tasks/index.js [hunk 1]
```diff
@@ -1,25 +1,39 @@
...
+  const [idTask, setIdTask] = useState('');
+
   useEffect(async () => {
     try {
       const response = await fetch(`${process.env.REACT_APP_API_URL}/tasks`);
-      const data = await response.json();
-      console.log(data);
-      setList(data.data);
+      const responseJson = await response.json();
+      // console.log(responseJson.data);
```
Comment (laura-brussa12) [inline src/Components/Tasks/index.js:15]: Acá en la línea 15 deberías quitar este comentario

File: src/Components/Tasks/index.js [hunk 2]
```diff
@@ -1,25 +1,39 @@
...
+      setList(responseJson.data);
     } catch (error) {
       console.error(error);
     }
   }, []);
+  const deleteItem = (id) => {
+    setIsOpen(true);
+    setIdTask(id);
+    console.log(idTask);
```
Comment (laura-brussa12) [inline src/Components/Tasks/index.js:24]: El console.log de la línea 24 de no ser necesario deberías sacarlo también

File: src/Components/Tasks/List/ListItem/Index.jsx [hunk 3]
```diff
@@ -2,11 +2,13 @@ import React from 'react';
 import styles from './listItem.module.css';
 
 const ListItem = ({ listItem, deleteItem }) => {
-  console.log('props', listItem);
+  // console.log('props', listItem);
```
Comment (laura-brussa12) [inline src/Components/Tasks/List/ListItem/Index.jsx:5]: Deberías quitar este comentario de la línea 5

File: src/Components/Layout/index.js [hunk 4]
```diff
@@ -30,6 +31,9 @@ function Layout() {
     case '/tasks':
       currentScreen = <Tasks />;
       break;
+    case '/task':
```
Comment (GuidoCerioni) [inline src/Components/Layout/index.js:34]: I think you souldn't change the URL, keep it '/tasks' and for the form, '/tasks/form'

File: src/Components/Task/index.jsx [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import styles from './Task.module.css';
+
+const Form = () => {
```
Comment (GuidoCerioni) [inline src/Components/Task/index.jsx:4]: You are not using this component on this branch i think

**Review by GuidoCerioni — COMMENTED** [review]
I think you have to make some changes, but im not sure about it. If im wrong please let me know. Btw, nice work Andrés!

**Comment by tobiaspujol** [comment]
I can't see the tasks list, and also can't add a task. In addition, when I refresh the website, it turns all white whit no data at all.

File: src/Components/Tasks/List/ListItem/Index.jsx [hunk 6]
```diff
@@ -0,0 +1,29 @@
...
+  const handleDelete = () => {
+    deleteItem(listItem._id);
+  };
+  let pencil = '✎';
+  let del = '✘';
+  return (
+    <tr className={styles.rows}>
+      <td>{listItem._id}</td>
+      <td>{listItem.description}</td>
+      <td>{listItem.projectId.name}</td>
```
Comment (Facund0Fernandez) [inline src/Components/Tasks/List/ListItem/Index.jsx:14]: Checking the branch I got the following error: 'TypeError: listItem.projectId is null' on this line

**Review by Facund0Fernandez — CHANGES_REQUESTED** [review]

File: src/Components/Tasks/List/ListItem/Index.jsx (hunk 6 above)
Comment (Gearyandres94) [reply src/Components/Tasks/List/ListItem/Index.jsx:14]: this happens because someone delete the project that the task connects to it with the populate.

**Review by Facund0Fernandez — APPROVED** [review]
Everything works correctly. Good job Andy!

**Pull request merged**